samples/statbar/statbar.cpp

```diff
@@ -966,24 +967,7 @@ MyStatusBar::~MyStatusBar()
 
 void MyStatusBar::OnSize(wxSizeEvent& event)
 {
-#if wxUSE_CHECKBOX
-    if ( !m_checkbox )
-        return;
-#endif
-
     wxRect rect;
-    if (!GetFieldRect(Field_Checkbox, rect))
-    {
-        event.Skip();
-        return;
-    }
-
-#if wxUSE_CHECKBOX
-    wxRect rectCheck = rect;
-    rectCheck.Deflate(2);
```

Comment:
It's unfortunate that there is no comment telling us why this call is needed in the first place. but testing (with and without) this `Deflate(2)` I didn't see any difference at all under Windows 10 or Linux, except under Gnome I got a warning which can be silenced by just removing this call which I did here.

Reply:
I'm pretty sure it was added because "it looked better" with it, but let's drop it if it's not needed any longer (in fact, I'd avoid changing the size of checkbox at all and just center it in the available space instead).